IRremoteESP8266.cpp

```diff
@@ -702,23 +702,32 @@ static void ICACHE_RAM_ATTR read_timeout(void *arg __attribute__((unused))) {
 
 static void ICACHE_RAM_ATTR gpio_intr() {
   uint32_t gpio_status = GPIO_REG_READ(GPIO_STATUS_ADDRESS);
+  uint32_t now = system_get_time();
```

Comment:
#TIL So reading this: http://bbs.espressif.com/viewtopic.php?t=1183#p3964 it appears using the micros() function should be executing from iram, thus faster (or the same as the system_get_time() ) so we should probably stick with calling micros(), at the very least for consistency and only a single thing to change for all occurrences etc.